tests/integration/targets/azure_rm_adapplication/tasks/main.yml

```
@@ -103,5 +120,4 @@
 
 - name: Assert there is no application
   ansible.builtin.assert:
-    that:
-      - output.applications | length == 0
+    that: output.applications | length == 0
```

No new line character at the end of file

---

This error still exist, No new line character at the end of file
```suggestion
    that:
      output.applications | length == 0
```